Dockerfile

```diff
@@ -0,0 +1,120 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I think it would still be nice for this to be in the /docker directory since this project does produce a number of docker images.